samples/helloworld-kt/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.5.10"
+    id("com.github.johnrengelman.shadow") version "7.0.0"
+}
+
+group = "alexa-skills-kit-samples"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("com.amazon.alexa:ask-sdk:2.40.0")
+    implementation("org.apache.logging.log4j:log4j-core:2.8.2")
```

Comment:
This is important `2.17.1` because of Log4j bug

Reply:
bumped up. Thanks for checking on this.